content/navigation.tsx

```diff
@@ -3,8 +3,6 @@ import React from 'react';
 import ProductIcon from '../src/components/ProductIcon';
 import { IItem } from '../src/types/Item';
 
-const year = 2022;
```

Comment:
I deleted this because it could be the case that a new GeoIP release notes page gets created before a new minFraud release notes page or vice versa.